src/methods/modular_programs/easy/q4/ReturnChange.java

```diff
@@ -0,0 +1,31 @@
+package methods.modular_programs.easy.q4;
+
+/*
+Write a Java Program that initializes three double variables and passes them to a method to return the number values
+after the decimal point. (e.g. if the double number is 14.5673, return 0.5673). If the variables are negative, return
+the negative decimal values.
```

Comment:
Change the last line of the question to:
`If the variables are negative, the returned value should be negative. `